style.css

```diff
@@ -0,0 +1,94 @@
+/*
+Primary:#81d3f9
+Primary-Light:#b5ffff
+Primary-Dark:#4ba2c6
+*/
+
+html {
+  background-color: #dddddd;
+}
+
+body {
+  margin: 0 auto;
+
+  max-width: 1024px;
+
+  font-family: "Roboto", sans-serif;
```

Comment:
` font-family: "Roboto", sans-serif;`
would be great to add addition fallback for font, not only `sans-serif`